assets/src/components/v2/departures/departure_row.tsx

```diff
@@ -1,17 +1,27 @@
-import React from "react";
+import React, { ComponentType } from "react";
 
-import RoutePill from "Components/v2/departures/route_pill";
+import RoutePill, { Pill } from "Components/v2/departures/route_pill";
 import Destination from "Components/v2/departures/destination";
-import DepartureTimes from "Components/v2/departures/departure_times";
+import DepartureTimes, {
+  TimeWithCrowding,
+} from "Components/v2/departures/departure_times";
 import DepartureAlerts from "Components/v2/departures/departure_alerts";
 
-const DepartureRow = ({
+type DepartureRow = {
+  id: string;
+  route: Pill;
+  headsign: Destination;
+  times_with_crowding: TimeWithCrowding[];
+  // currently never used, will be removed
+  inline_alerts: any[];
```

Comment:
**question:** if this isn't used is there a reason to keep it in the type? passing additional properties to a react component should not raise a type error. then the `DepartureRow` component is explicit about only the things it _needs_.

<!-- i'm using https://conventionalcomments.org/ -->

Reply:
By "never used" I meant the server currently never returns anything in this field other than an empty array — it is still accessed in the frontend code (in this component, in fact), and there is a "property does not exist" error if this line is removed from the type. Because of this current "non-use" and expected future removal, and because it's somewhat outside the scope of the task, I didn't think it was important to fill in a detailed type for it. The wording of the comment could definitely be improved. 😅

Reply:
Ah okay. That makes sense. Thanks for the clarification!